src/main/java/mate/academy/Main.java

```diff
@@ -2,17 +2,29 @@
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
 import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
 import mate.academy.model.MovieSession;
+import mate.academy.model.User;
 import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
 import mate.academy.service.MovieSessionService;
+import mate.academy.service.UserService;
 
 public class Main {
+    private static final Injector injector
+            = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
-        MovieService movieService = null;
+        UserService userService = (UserService) injector.getInstance(UserService.class);
+        User user = new User();
+        user.setEmail("username@.com");
```

Comment:
The email format 'username@.com' is incorrect. It should be a valid email format, such as 'username@example.com'.